Chap_API_Key_Value_Mgmt.tex

```diff
@@ -128,19 +128,12 @@ \subsection{\code{PMIx_Get}}
 %%%%
 \descr
 
-Retrieve information for the specified \refarg{key} as published by the process identified in the given \refstruct{pmix_proc_t}, returning a pointer to the value in the given address.
+Retrieve information for the specified \refarg{key} associated with the process identified in the given \refstruct{pmix_proc_t}, returning a pointer to the value in the given address. In general, data posted by a process via \refapi{PMIx_Put} and data that refers directly to a process-related value must be retrieved by specifying the rank of the target process. All other information is retrievable using a rank of \refconst{PMIX_RANK_WILDCARD}, as illustrated in \ref{chap:api_kv:getex}. See \ref{api:struct:attributes:retrieval} for an explanation regarding use of the \emph{level} attributes, and \refapi{PMIx_server_register_nspace} for a description of the available information.
```

Comment:
For any "pre-populated" information that wasn't necessarily `PMIx_Put` by another process/rank, is there a section within the standard that lists the reserved keys for that info?  Is Section 3.4 (Reserved Attributes) that section (maybe 3.47 - 3.4.14 more specifically)?  If so, it might be good to cross-reference that.  After reading through just the PMIx_Get description, it isn't clear to me necessarily where I should go to learn more about what pre-populated information I can get via this interface (subject to implementation support of course).  And I think the  `api:struct:attributes:retrieval` section is just a subset of the possible pre-populated information, correct?

Reply:
By definition, info "put" by another process/rank can't be in the pre-populated info and therefore any such info cannot use a reserved key - I'll try to make that clearer and reference Section 3.4.

IIRC, I was intending `api:struct:attributes:retrieval` section to contain all _required_ pre-populated info - I'll double check and ensure it does, adding anything that is missing. I'll also be sure to add all the defined _optional_ info keys so it provides a complete picture.